usr/src/man/man1/ld.1

```diff
@@ -109,9 +109,7 @@
 .Op Fl z Cm sysroot= Ns Ar directory
 .Op Fl z Cm target= Ns Cm sparc Ns | Ns Cm x86 Ns | Ns Cm aarch64
 .Op Fl z Cm text | Fl z Cm textwarn | Fl z Cm textoff
-.Sm off
-.Op Fl z\~ Cm type= Cm exec | kmod | reloc | shared
-.Sm on
+.Op Fl z Cm type= Ns Cm exec Ns | Ns Cm kmod Ns | Ns Cm reloc Ns | Ns Cm shared
```

Comment:
I think I'm still missing the point here - here's how it renders in my terminal

<img width="301" alt="image" src="https://github.com/richlowe/illumos-gate/assets/29426693/9aaa5045-c939-47f9-adf3-ab121aaff8eb">

Reply:
I took the `Ns/Cm` bits from a prior example.  I deleted the `Sm` and `\~` because in my terminal it rendered like:

![image](https://github.com/richlowe/illumos-gate/assets/245797/ca3b11bd-937f-4bb9-b9ff-6bade0654ccb)